docs/requirements-doc-test.txt

```diff
@@ -1,2 +1,6 @@
 colossalai
 torch
+packaging
+tensornvme
+psutil
+transformers
```

Comment:
Should we add a command to tell the user to install these dependencies in the doc?

Reply:
ok

Reply:
like 

```bash
pip install packaging tensornvme psutil transformers
```